packages/block-library/src/embed/editor.scss

```diff
@@ -19,6 +19,10 @@
 
 	.components-placeholder__learn-more {
 		margin-top: 1em;
+		.components-external-link,
```

Comment:
We may have placeholders outside of the embed block. Would it make sense to have a generic rule for the "a" element inside the placeholder component in packages/components/src/placeholder/style.scss?

Reply:
@jorgefilipecosta okay
do means give style inside packages/components/src/placeholder/style.scss file, Right ?